### DISCUSSION TITLE
How to disable automigrations?

### DISCUSSION THREAD

**Discussion by denisgurev** [pocketbase/pocketbase#1369] [Q&A] [answered]
Automigrations are great, but sometimes I'm just experimenting with my collections and I don't want to create unnecessary files (I know that I could just delete them, but still).

Is there a way to disable the migrations creation temporary?

**Comment by ganigeorgiev** [comment, marked as answer]
Hm, it depends on the desired workflow, but you can start the executable with `--automigrate=0`.

Later, when you want to enable them back, you can remove the flag (or set it to `1`).
To ensure that all collection changes are in a migration and in case you decide to do other changes after that, it could be good idea also to run `./pocketbase migrate collections` to create a full snapshot of your latest changes so that the automigrate could be applied correctly from that point on.

**Reply by denisgurev** [reply]
Thank you and sorry for the silly question. I totally forgot about the flag.
